stingray/lightcurve.py

```diff
@@ -177,13 +186,14 @@ def __init__(self, time, counts, err=None, input_counts=True,
 
         self.err_dist = err_dist
 
-        self.tstart = self.time[0] - 0.5*self.dt
-        self.tseg = self.time[-1] - self.time[0] + self.dt
+        self.tstart = np.min(self.time) - 0.5*self.dt
```

Comment:
Can we then set some kind of switch, with the light curve making faster operations under a specific set of conditions? (ordered counts etc.)
`sort_counts()` might be used in some specific cases (looking for bursts, maybe), but the vast majority of users will  probably just want a standard, time-ordered light curve